modules/queue-manager/variables.tf

```diff
@@ -21,7 +21,7 @@ variable "name" {
 variable "queue_manager_version" {
   description = "The MQ version of the queue manager."
   type        = string
-  default     = "9.4.0_3"
+  default     = "9.4.1_2"
```

Comment:
@shemau We will need to find a way to automate the management of this version. Have you thought about it yet? I created an issue with some proposals: https://github.com/terraform-ibm-modules/terraform-ibm-mq-cloud/issues/82
Let me know what you think